netlogo-gui/src/main/lab/gui/Supervisor.scala

```diff
@@ -74,21 +75,31 @@ class Supervisor(dialog: java.awt.Dialog,
       val fileName = org.nlogo.swing.FileDialog.showFiles(
         workspace.getFrame, "Exporting as spreadsheet", java.awt.FileDialog.SAVE,
         workspace.guessExportName(worker.protocol.name + "-spreadsheet.csv"))
-      addExporter(new SpreadsheetExporter(
+      try{
+	    addExporter(new SpreadsheetExporter(
         workspace.getModelFileName,
         workspace.world.getDimensions,
         worker.protocol,
```

Comment:
Make sure these statements are indented one level in from the `addExporter` start.